### ISSUE TITLE
About BBD 100K challenge

### ISSUE THREAD

**Issue by yueruchen** [qijiezhao/CFENet#4]
Hi, thanks for your novel work.

I saw your model award 2nd price on WAD 2018 challenge on BBD 100k dataset. Will you release the model? Actually I am not working on detection problem but we need some detection model to help us on another task. Your work will be very helpful to us and we will cite your paper.

**Comment by qijiezhao** [comment]
@yueruchen Thanks, the model is not suitable for this code. i.e., this repo is actually an updated version of CFENet while the pre-trained model on BDD is an old one. 
But we still have a detection result file(.json) generated by the model, if you need the detection results on BDD dataset, we are glad to provide it.

**Comment by yueruchen** [comment]
@qijiezhao Hi, thanks for your reply.

Actually we need the pre-trained model to test on the validation set on BBD 100k. We will use the detection metric to evaluate our method. Could please help with that? If it is not convenient to upload here you can send it to my email yifanjiang97@gmail.com.

Thanks so much!
